.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,56 @@
-<!--- Prepend PR title with [WIP] if work in progress. Remove when ready for review. -->
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
+  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
 
-<!--- For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments -->
+  Before submitting a Pull Request, please ensure you've done the following:
+  - 📖 Read the Contributing Guide: https://github.com/codebuddies/frontend/blob/main/CONTRIBUTING.md.
+  - 📖 Read the Open Sauced Code of Conduct: https://github.com/codebuddies/frontend/blob/main/CODE_OF_CONDUCT.MD.
+  - 👷‍♀️ Aim to create small PRs by splitting up feature work, and potentially multiple issues.
+  - ✅ Provide tests for your changes if relevant.
+  - 📝 Use descriptive commit messages.
+  - 📗 Update any related documentation and include any relevant screenshots.
+  Note: h/t to https://github.com/open-sauced/open-sauced/blob/main/.github/PULL_REQUEST_TEMPLATE.md for this pull request template model.
+-->
 
 ## What type of PR is this? (check all applicable)
 
-- [ ] Refactor
-- [ ] Feature
-- [ ] Bug Fix
-- [ ] Documentation Update
+- [ ] ♻️ Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 🎨 Enhancement
+- [ ] 📝 Documentation Update
+- [ ] 🔖 Release
+- [ ] 🚩 Other
 
-## Description
+## Context
 
-## Related Tickets & Documents
+Placeholder example: This PR closes issue #12
 
-## Before & After
+## Screenshots/Recordings (if there are UI changes)
+
+Placeholder example: ![screenshot of new resource page](https://user-images.githubusercontent.com/4512699/89255710-3a3d4400-d5d7-11ea-9d56-8be1271d77a2.png)
+
+## Implementation Details - what was your thought process as you changed the code?
+
+Placeholder example:
+[x] Made an axios GET request to /resources
+[x] Refactored to use useQuery library
+[x] Wrote tests
+
+## Related Tickets & Documents (Optional)
+
+Placeholder example: This relates to issue #23.
+
+Filed issue #43 as a next step to do after this PR is merged.
+
+## Added tests?
+
+- [ ] 👍 yes
+- [ ] 🙅 no, because they aren't needed
+- [ ] 🙋 no, because I need help
+
+## Added to documentation?
```

Comment:
@lpatmo clarified that documentation refers to the contributing.md right now.  I thought that "documentation" might refer to storybook.

Reply:
Good call-out... will update this section to clarify what we mean by "documentation"

Reply:
Made the change in 28030f1.